grblas/tests/test_numpyops.py

```diff
@@ -177,19 +177,20 @@ def test_npmonoid():
             [np.array([True, False, True, False]), np.array([True, True, False, False])],
         ],
     ]
-    if _supports_complex:  # pragma: no branch
-        data.append(
-            [
-                [
-                    grblas.Vector.from_values(index, values1, dtype="FC64"),
-                    grblas.Vector.from_values(index, values2, dtype="FC64"),
-                ],
-                [
-                    np.array(values1, dtype=np.complex128),
-                    np.array(values2, dtype=np.complex128),
-                ],
-            ]
-        )
+    # Complex monoids not working yet (they segfault upon creation in grblas.operators)
+    # if _supports_complex:  # pragma: no branch
+    #     data.append(
+    #         [
+    #             [
+    #                 grblas.Vector.from_values(index, values1, dtype="FC64"),
+    #                 grblas.Vector.from_values(index, values2, dtype="FC64"),
+    #             ],
+    #             [
+    #                 np.array(values1, dtype=np.complex128),
+    #                 np.array(values2, dtype=np.complex128),
+    #             ],
+    #         ]
+    #     )
```

Comment:
Btw @jim22k complex monoids still don't work.  I apparently added a check in "operators.py" to avoid the segfault.